src/main/sensors/battery.c

```diff
@@ -225,10 +226,6 @@ void batteryUpdatePresence(void)
         batteryWarningVoltage = 0;
         batteryCriticalVoltage = 0;
     }
-    if (debugMode == DEBUG_BATTERY) {
```

Comment:
This should not be removed - if new debug values need to be added they should get a new debug mode.

Reply:
I don't think anyone actively uses the last two `BATTERY` debug values at present.  In blackbox explorer they show up as 'Unused'.  They were probably included for debugging purposes when developing the original battery code, and are really not required anymore.  The two I re-purposed were `batteryCellCount`, which is readily identified without resort to a debug, and `isVoltageStable`, a value that is set early in startup, and not relevant to the end user.
This code only really needs two debug values for user display purposes.
I thought that since this is a 'BATTERY` related function, it would be ideal if they could be in the `BATTERY`debug, where people would most readily find them.
And since there are two parameters that show up as unused in `BATTERY`, it seemed like a good idea to use them for the two parameters needed here.
I can definitely make a separate debug mode, but am a bit reluctant to do so, as I wouldn't make use of it fully.

Reply:
@mikeller I have done an update, but in it, I left the debug unchanged, please give consideration to keeping it as is for user level simplicity.